File: python_project/Arrays/LinearSearch.py
a=[3,4,5,6,7,5,4,3]
target = 5
firstIndex= -1
lastIndex = -1
for i in range(len(a)):
    if a[i]== target:
        if firstIndex ==-1:
            firstIndex=i
        lastIndex = i
print(target, " is the target")
print("First index: " + str(firstIndex))
print("Last index: " + str(lastIndex))

File: js_project/Array/LinearSearch.js
let a= [3,4,5,6,7,5,4,3];
let target = 5;
let firstIndex = -1;
let lastIndex = -1;
for (let i=0; i<a.length; i++){
    if (a[i]==target){
        if (firstIndex==-1){
            firstIndex= i;
        }
        lastIndex=i;
    }
}
console.log(target+" is the target number");
console.log (firstIndex);
console.log(lastIndex);